car_description/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>car_description</name>
+    <version>1.0.0</version>
+    <description>The car_description package</description>
+    <maintainer email="alex.bordallo@ed.ac.uk">Alejandro Bordallo</maintainer>
```

Comment:
Could you please add me as maintainer? :)

Reply:
Done for all tartan_carpet repos